humble/distribution.yaml

```diff
@@ -2730,13 +2730,10 @@ repositories:
       url: https://gitlab.com/ApexAI/performance_test.git
       version: 1.1.0
     release:
-      packages:
-      - performance_report
```

Comment:
@revati-naik IIUC this means `performance_report` will no longer be released. Is that intended?